tests/complexity.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var krabbyTest = require('./_baseTest');
+var escomplex = require('escomplex-js');
+var _ = require('lodash');
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+
+var baseConfig = {
+  testConfig: {
+  },
+  files: ['./**/*.js', '!./node_modules/**/*.js', '!./**/node_modules/**/*.js']
+};
+
+var ComplexityTest = new krabbyTest(baseConfig);
+
+ComplexityTest.prototype.analyze = function(files) {
+  var self = this;
+
+  return Promise.map(files, function(fileName) {
+    return fs.readFileAsync(fileName)
```

Comment:
haha, simpler apis make for simpler testers. 

```
async.map(files, fs.readFile, function(err, results) {
  if (err) return console.warn(fileName, e);

  var result = escomplex.analyse(data.toString(), self.config.testConfig);
  self.logs.logs.push(result);
  return result;
});
```

![supa hot fire meme reaction gif](http://i.imgur.com/xgz9nkR.gif)